packages/frontend/src/components/CustomVisualization/index.tsx

```diff
@@ -20,18 +21,24 @@ const CustomVisualization: FC<Props> = (props) => {
     const { isLoading, visualizationConfig, resultsData } =
         useVisualizationContext();
 
+    const [ref, rect] = useResizeObserver();
+
     useEffect(() => {
         // Load all the rows
         resultsData?.setFetchAll(true);
     }, [resultsData]);
 
-    if (isLoading) {
-        return <LoadingChart />;
-    }
-
     if (!isCustomVisualizationConfig(visualizationConfig)) return null;
     const spec = visualizationConfig.chartConfig.validConfig.spec;
 
+    if (isLoading) {
+        return (
+            <Center h="100%">
```

Comment:
There is an issue with the loading height
![loading height](https://github.com/user-attachments/assets/a143e9a1-5efc-4c30-a5ad-092b9e2b1675)

Reply:
im just gonna keep the loading as is